From 5df7c109ccd7e79f5e18bf1dc7247381cfa8dcc6 Mon Sep 17 00:00:00 2001
From: Ethan Cotterell <ethancotterell@gmail.com>
Date: Mon, 18 Jan 2021 12:20:23 +0000
Subject: [PATCH] include: removed dependency on sys/epoll.h

---
 libparistraceroute/os/sys/epoll.h | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/libparistraceroute/os/sys/epoll.h b/libparistraceroute/os/sys/epoll.h
index 24f6f9b..4cda3db 100644
--- a/libparistraceroute/os/sys/epoll.h
+++ b/libparistraceroute/os/sys/epoll.h
@@ -3,14 +3,14 @@
 
 #include "../os.h"
 
-#ifdef LINUX
-#  include <sys/epoll.h>
-#endif
+// #ifdef LINUX
+// #  include <sys/epoll.h>
+// #endif
 
-#ifdef FREEBSD
-#  include "eventpoll.h"
-#  include <stdio.h>
-#  include <stdint.h>
+// #ifdef FREEBSD
+#include "eventpoll.h"
+#include <stdio.h>
+#include <stdint.h>
 
 typedef union epoll_data
 {
@@ -64,6 +64,6 @@ int epoll_create1(int flags);
 int epoll_ctl(int epfd, int op, int fd, struct epoll_event *event);
 int epoll_wait(int epfd, struct epoll_event *events, int maxevents, int timeout);
 
-#endif
+// #endif
 
 #endif // OS_SYS_EPOLL
-- 
2.25.1

